lesson_16/part_1/employee.py

```diff
@@ -0,0 +1,35 @@
+from abc import ABC
+
+
+class Employee(ABC):
+
+    name: str
```

Comment:
such type of attributes is better to store as private if you do not want to provide access to them from any place of your program.